packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -331,6 +340,33 @@ class IOSDevice extends Device {
 
   @override
   Future<Null> takeScreenshot(File outputFile) => iMobileDevice.takeScreenshot(outputFile);
+
+  // Maps stdout line stream. Must return original line.
+  String monitorInstallationFailure(String stdout) {
+    // Installation issues.
+    if (stdout.contains('Error 0xe8008015') || stdout.contains('Error 0xe8000067')) {
+      printError(noProvisioningProfileInstruction, emphasis: true);
+
+    // Launch issues.
+    } else if (stdout.contains('e80000e2')) {
```

Comment:
Maybe add the '0x'?

Reply:
ios_deploy is pretty inconsistent. In this case, it doesn't actually print the 0x